muxers/mplex/src/lib.rs

```diff
@@ -100,7 +99,7 @@ impl MplexConfig {
     #[inline]
     fn upgrade<C>(self, i: C, endpoint: Endpoint) -> Multiplex<C>
     where
-        C: AsyncRead + AsyncWrite
+        C: AsyncRead + AsyncWrite,
```

Comment:
I do not get why rustfmt sometimes wants to add unnecessary commas and at other times it removes them.